### DISCUSSION TITLE
Is it possible to use flutter_data library in a FlutterFlow project ?

### DISCUSSION THREAD

**Discussion by loureirodnl** [flutterdata/flutter_data#284] [Q&A]
Hello all !

We want to create a health app using FlutterFlow. All the data will be handled by a REST API backend (FHIR server for those who are familliar with health industry). The app need to be compatible with offline usage because it will be used in a humanitarian context. For this reason flutter_data seems a perfect fit. However we need to use FlutterFlow (Low-Code app builder) to build the app.

Any people here already tried to implement flutter_data in a FlutterFlow project ? Or maybe you can just pinpoint me the reasons why it won't be feasible ?

Thank you very much for your help !

**Comment by frank06** [comment]
Hello! Sorry I have no idea
